src/rucksack-1-freeimage.patch

```diff
@@ -0,0 +1,109 @@
+@@ -80,65 +78,30 @@ set_target_properties(rucksack_static PROPERTIES
+   OUTPUT_NAME rucksack
+   COMPILE_FLAGS ${LIB_CFLAGS})
+ 
+-add_library(rucksack_shared SHARED ${RUCKSACK_LIB_SOURCES} ${RUCKSACK_LIB_HEADERS})
+-set_target_properties(rucksack_shared PROPERTIES
+-  OUTPUT_NAME rucksack
+-  SOVERSION ${VERSION_MAJOR}
+-  VERSION ${VERSION}
+-  COMPILE_FLAGS ${LIB_CFLAGS})
+-
+ 
+-include_directories(${FreeImage_INCLUDE_DIRS})
++include_directories(${FREEIMAGE_INCLUDE_DIRS})
+ add_library(rucksackspritesheet_static STATIC
+   ${RUCKSACK_SPRITESHEET_LIB_SOURCES} ${RUCKSACK_SPRITESHEET_LIB_HEADERS})
+ set_target_properties(rucksackspritesheet_static PROPERTIES
+   OUTPUT_NAME rucksackspritesheet
+   COMPILE_FLAGS ${LIB_CFLAGS})
+ 
+-add_library(rucksackspritesheet_shared SHARED
+-  ${RUCKSACK_SPRITESHEET_LIB_SOURCES} ${RUCKSACK_SPRITESHEET_LIB_HEADERS})
+-set_target_properties(rucksackspritesheet_shared PROPERTIES
+-  OUTPUT_NAME rucksackspritesheet
+-  SOVERSION ${VERSION_MAJOR}
+-  VERSION ${VERSION}
+-  COMPILE_FLAGS ${LIB_CFLAGS})
+-target_link_libraries(rucksackspritesheet_shared ${FreeImage_LIBRARIES})
+-
+-
+ 
+ add_executable(rucksack ${EXE_SOURCES} ${EXE_HEADERS})
+-target_link_libraries(rucksack rucksack_shared rucksackspritesheet_shared ${LAXJSON_LIBRARY})
++target_link_libraries(rucksack rucksack_static rucksackspritesheet_static ${LAXJSON_LIBRARY} ${FREEIMAGE_LIBRARIES})
+ include_directories(${LAXJSON_INCLUDE_DIR})
+ set_target_properties(rucksack PROPERTIES
+   COMPILE_FLAGS ${EXE_CFLAGS})
+ 
+ install(TARGETS rucksack DESTINATION bin)
+ install(TARGETS
+-  rucksack_static rucksack_shared
+-  rucksackspritesheet_static rucksackspritesheet_shared
++  rucksack_static
++  rucksackspritesheet_static
+   DESTINATION lib)
+ install(FILES "src/rucksack.h" DESTINATION include/rucksack)
+ install(FILES "src/spritesheet.h" DESTINATION include/rucksack)
+ 
+ 
+-enable_testing()
+-include_directories("${PROJECT_SOURCE_DIR}/src")
+-add_executable(test_library test/test_library.c)
+-set_target_properties(test_library PROPERTIES
+-  COMPILE_FLAGS ${EXE_CFLAGS})
+-target_link_libraries(test_library rucksack_shared rucksackspritesheet_shared)
+-add_test(LibraryTests test_library)
+-
+-add_executable(test_path test/test_path.c src/path.c src/path.h)
+-set_target_properties(test_path PROPERTIES
+-  COMPILE_FLAGS ${EXE_CFLAGS})
+-add_test(PathUnitTests test_path)
+-
+-add_executable(test_stringlist test/test_stringlist.c src/stringlist.c src/stringlist.h)
+-set_target_properties(test_stringlist PROPERTIES
+-  COMPILE_FLAGS ${EXE_CFLAGS})
+-add_test(StringListTests test_stringlist)
+-
+ message("\n"
+ "Installation Summary\n"
+ "--------------------\n"
+@@ -152,7 +115,7 @@ message(
+ "System Dependencies\n"
+ "-------------------\n"
+ "* C99 Compiler                 : ${STATUS_C99}\n"
+-"* freeimage                    : ${STATUS_FREEIMG}\n"
++"* freeimage                    : patched\n"
```

Comment:
You can use FREEIMAGE_FOUND. See http://www.cmake.org/cmake/help/v3.0/module/FindPkgConfig.html